modules/apps/blogs/blogs-demo/src/main/java/com/liferay/blogs/demo/internal/BlogsDemo.java

```diff
@@ -139,11 +125,14 @@ private <T> T _getRandomElement(List<T> list) {
 
 	private final List<BlogsEntryDemoDataCreator> _blogsEntryDemoDataCreators =
```

Comment:
@JanisZhang It's weird that we reference fixed number of `BlogsEntryDemoDataCreator` services, put them into a list, and use them by iterating the list. Currently there are only 2 of them and both are referenced statically, so let's remove the list field, and call these `BlogsEntryDemoDataCreator` instances directly.

If in the future they want more to be added, they should use something like service tracker list.